desktop/apps/fairs/query.coffee

```diff
@@ -27,6 +27,7 @@ module.exports = '''
       }
       location {
```

Comment:
I wonder if there's an opportunity to output a formatted string from Metaphysics for this (e.g. in case Eigen needs this fall-back logic as well).